test/util/create-ssb.js

```diff
@@ -11,10 +11,14 @@ const caps = { shs: crypto.randomBytes(32).toString('base64') }
 
 const randomName = () => crypto.randomBytes(16).toString("hex");
 
-module.exports = function createSSB (name = randomName(), opts = {}) {
+module.exports = function createSSB (name = randomName(), opts = {}, plugins) {
```

Comment:
Now `createSSB()` can add plugins! I'm using this to test SSB-Private2.

Reply:
I think we've accidentally recreated `scuttle-testbot` ...?

Reply:
oh, except that thing has `ssb-db` as a dep

Reply:
Oh yeah, it's super similar. I think the SSB-DB dep would make it hard to test in this module, since we want to load our own SSB-DB, but that's good to know for the future.